### PR TITLE
added db to html

### PR DESCRIPTION
**Pull request by JVLoe**
I added the db to my html - only in computer readable format - next pull for human readable

### CODE REVIEW

File: collectorApp.php [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+$dbUsername = 'root';
+$dbPassword = 'password';
+$db = new PDO($connectionString, $dbUsername, $dbPassword);
+
+$queryString = 'SELECT * FROM `myplanets`;';
+$query = $db->prepare($queryString);
+$query->execute();
+
+$allResults = $query->fetchAll();
+$firstResult = $query->fetch();
```
Comment (Harry-Johnston) [inline collectorApp.php]: do you need both an $allResults variable and $firstResult

File: collectorApp.php [hunk 2]
```diff
@@ -1,5 +1,19 @@
 <?php
 
+$connectionString = 'mysql:host=db; dbname=planetdatabase';
+$dbUsername = 'root';
+$dbPassword = 'password';
+$db = new PDO($connectionString, $dbUsername, $dbPassword);
+
```
Comment (Harry-Johnston) [inline collectorApp.php:7]: might be worth adding the below line

$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);

**Review by Harry-Johnston — CHANGES_REQUESTED** [review]
Only change i would make is to add the below code  after you create the db. This formats the output array as a associative array and i think is easy to use

$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);

File: collectorApp.php (hunk 1 above)
Comment (JVLoe) [reply collectorApp.php]: Thanks :)

Comment (JVLoe) [reply collectorApp.php]: its more to see if it works, will delete before main

**Review by Harry-Johnston — APPROVED** [review]

**Pull request merged**